### PR TITLE
fix(helper): cached_property decorator fixed

### PR DESCRIPTION
**Pull request by rohan1chaudhari**
cached_property decorator was not retrieving from the right key

skip

### CODE REVIEW

**Comment by JoanFM** [comment]
recheckcla

**Comment by rohan1chaudhari** [comment]
> **Jina CLA check**
> 
>  Thank you for your pull request. It looks like this is your first contribution to an open source project maintained by Jina AI Limited. Before we can look at your pull request, we kindly ask that you sign our [Contributor License Agreement](https://github.com/jina-ai/jina/blob/master/CLA.md). You can sign it by commenting in format below.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_


I have read the CLA Document and I hereby sign the CLA

**Comment by JoanFM** [comment]
recheckcla

**Comment by rohan1chaudhari** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by JoanFM** [comment]
recheckcla

File: jina/helper.py [hunk 1]
```diff
@@ -666,13 +666,14 @@ def _getattr(obj, attr):
...
 
-class cached_property:
+class cached_property(object):
     def __init__(self, func):
         self.func = func
 
     def __get__(self, obj, cls):
-        if obj is None:
-            return self
+        cached_value = obj.__dict__.get(f'CACHED_{self.func.__name__}')
```
Comment (hanxiao) [inline jina/helper.py:674]: you should use get(, None) so that this never gives a key error

**Pull request merged**